### PR TITLE
[Intro] Add dinosaur illustration to code of ethics

### PR DESCRIPTION
**Pull request by ehumph**
Because we thought the image of a dinosaur with an angel and a devil on its shoulders would be cute.

### CODE REVIEW

**Comment by ehumph** [comment]
Resolves #54

File: 01-intro.Rmd [hunk 1]
```diff
@@ -52,6 +52,10 @@ In summary, while AI chatbots can provide valuable assistance to software develo
 
 Our first duty as programmers is to avoid causing harm to people and the world. This includes avoiding both intentional and inadvertent harm. *Intentional harm* is harm that is caused by a programmer's deliberate actions. For example, a programmer might create a virus that is designed to harm computers. *Inadvertent harm* is harm that is caused by a programmer's negligence or lack of foresight. For example, a programmer might create a program that is buggy and causes data loss.
 
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The dinosaur cartoon says in a speech bubble 'Making the ethical choice requires careful thought!' while thinking about an angel and a devil, which represent good and bad choices."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MCNeSO4aOm1iESWDLOGTcx3aLEbnu8UttV0QGVAeafE/edit#slide=id.g2313b08e75a_243_0")
```
Comment (cansavvy) [inline 01-intro.Rmd:56]: This is amazing! Can we make the devil and angel not just thought bubbles but in reality?

**Review by cansavvy — APPROVED** [review]
LGTM!

**Comment by avahoffman** [comment]
Love it so much 🤩

**Review by carriewright11 — APPROVED** [review]
YAY love it :)

**Pull request merged**